File: run_git.js
const { execSync } = require('child_process');
const path = require('path');

const repoPath = 'C:/Users/PC/claude-launcher-web';

try {
  console.log('=== git status ===');
  const status = execSync('git status', { cwd: repoPath, encoding: 'utf8' });
  console.log(status);
} catch (err) {
  console.log('git status error:', err.message);
}

try {
  console.log('\n=== git diff server.js ===');
  const diff = execSync('git diff server.js', { cwd: repoPath, encoding: 'utf8' });
  console.log(diff);
} catch (err) {
  console.log('git diff error:', err.message);
}

File: run_git.py
import subprocess
import os

os.chdir(r'C:\Users\PC\claude-launcher-web')

# Run git status
result = subprocess.run(['git', 'status'], capture_output=True, text=True)
print("=== git status ===")
print(result.stdout)
print(result.stderr)

# Run git diff server.js if there are changes
result = subprocess.run(['git', 'diff', 'server.js'], capture_output=True, text=True)
print("\n=== git diff server.js ===")
print(result.stdout)
print(result.stderr)
